keadm/cmd/keadm/app/cmd/helm/cloudcore.go

```diff
@@ -90,21 +90,9 @@ func NewCloudCoreHelmTool(kubeConfig, kubeedgeVersion string) *CloudCoreHelmTool
 
 // Install uses helm client to install cloudcore release
 func (c *CloudCoreHelmTool) Install(opts *types.InitOptions) error {
-	externValueFile := ""
+	var externValueFile string
```

Comment:
The externValueFile variable seems to be of little use, and you can use opts.Profile directly.

Reply:
ok